code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1224,13 +1230,22 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 
 		if(user.limb_destroyer)
 			target.dismembering_strike(user, affecting.body_zone)
-		target.apply_damage(damage, attack_type, affecting, armor_block)
-		log_combat(user, target, "punched")
+
+		if(atk_verb == ATTACK_EFFECT_KICK)//kicks deal 1.5x raw damage
+			target.apply_damage(damage*1.5, attack_type, affecting, armor_block)
+			log_combat(user, target, "kicked")
+		else//other attacks to 1/2 raw damage + full in stamina damage
+			target.apply_damage(damage/2, attack_type, affecting, armor_block)
+			target.apply_damage(damage, STAMINA, affecting, armor_block)
+			log_combat(user, target, "punched")
+
 		if((target.stat != DEAD) && damage >= user.dna.species.punchstunthreshold)
 			target.visible_message("<span class='danger'>[user] has knocked  [target] down!</span>", \
 							"<span class='userdanger'>[user] has knocked [target] down!</span>", null, COMBAT_MESSAGE_RANGE)
-			target.apply_effect(80, EFFECT_KNOCKDOWN, armor_block)
+			var/knockdown_duration = 40 + (target.getStaminaLoss() + (target.getBruteLoss()/2))/1.25 //50 total damage = 40 base stun + 40 stun modifier = 80 stun duration, which is the old base duration
```

Comment:
```suggestion
			var/knockdown_duration = 40 + (target.getStaminaLoss() + (target.getBruteLoss()*0.5))*0.8 //50 total damage = 40 base stun + 40 stun modifier = 80 stun duration, which is the old base duration
```

Reply:
Why? Seems less readable

Reply:
`Do not divide when you can easily convert it to multiplication. (ie 4/2 should be done as 4*0.5)`